steam_wrapper/steam_wrapper.py

```diff
@@ -266,10 +272,14 @@ def main(steam_binary=STEAM_PATH):
     logging.info("https://github.com/flathub/com.valvesoftware.Steam/wiki")
     current_info = read_flatpak_info(FLATPAK_INFO)
     check_allowed_to_run(current_info)
-    migrate_config()
-    migrate_data()
+    should_restart = migrate_config()
+    should_restart += migrate_data()
     migrate_cache()
-    timezone_workaround()
-    configure_shared_library_guard()
-    enable_discord_rpc()
-    os.execv(steam_binary, [steam_binary] + sys.argv[1:])
+    if should_restart:
+        command = ["/usr/bin/flatpak-spawn"] + sys.argv
```

Comment:
Better log that we're going to restart the app.

Reply:
Added logging